nle/tests/test_envs.py

```diff
@@ -365,7 +365,7 @@ def test_final_reward(self, env):
 
         # Hack to quit.
         env.env.step(nethack.M("q"))
-        _, reward, done, _ = env.step(env._actions.index(ord("y")))
+        obs, reward, done, _ = env.step(env._actions.index(ord("y")))
```

Comment:
Why this change?

Reply:
i was undoing a previous change to the tests a few commits back, and slipped.

Reply:
```suggestion
        _, reward, done, _ = env.step(env._actions.index(ord("y")))
```